src/Snppts/Snippets/Enlighten.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Snppts.Authors;
+using Snppts.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Snppts.Snippets
+{
+    
+    public class Enlighten : IAmASnippet
+    {
+        public string Slug => "Enlighten";
+        public string Title => "Enlighten";
+        public string GithubRepoName => "devoirtechsandip/SnpptsChallenge";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
```

Comment:
I think this is Android only, right?

Reply:
Yes, and I have changed the file accordingly